docs/api-reference/oembed.md

```diff
@@ -17,10 +17,10 @@ Parameters:
 _Example URLs:_
 
 - Jovian Notebook URLs
-  - [https://jovian.ml/aakashns/01-pytorch-basics](https://jovian.ml/aakashns/01-pytorch-basics)
-  - [https://jovian.ml/aakashns/movielens-fastai/v/14](https://jovian.ml/aakashns/movielens-fastai/v/14)
+  - [Jovian Notebook](https://jovian.ml/aakashns/01-pytorch-basics)
+  - [Jovian Notebook (Version specified)](https://jovian.ml/aakashns/movielens-fastai/v/14)
 - Jovian Viewer URLs
-  - [http://jovian.ml/viewer?url={notebook-url}](http://jovian.ml/viewer?url=https%3A%2F%2Fjvn.storage.googleapis.com%2Fgists%2Faakashns%2F5bc23520933b4cc187cfe18e5dd7e2ed%2Fraw%2F901a9d2508bd441dbf06954c5f46bf58%2Fmovielens-fastai.ipynb)
+  - [https://jovian.ml/viewer?url={notebook-url}](https://jovian.ml/viewer?url=https%3A%2F%2Fjvn.storage.googleapis.com%2Fgists%2Faakashns%2F5bc23520933b4cc187cfe18e5dd7e2ed%2Fraw%2F901a9d2508bd441dbf06954c5f46bf58%2Fmovielens-fastai.ipynb)
```

Comment:
add text in []

Reply:
That label for the link is a example of the URL format to be used, think it can be retained as above. Let me know if it needs to be changed.